### ISSUE TITLE
Not sure how to get working

### ISSUE THREAD

**Issue by Alda2308** [GHolli/eBUS-control#1]
Hi,
I have ebus adapter v2.0 produced and Vaillant VRC700/6. I tried connect directly via rs232/usb adapter dirctly into my HW where openhab is running. Problem is, that VRC700 wasn't detected as VRC700 but as only standard device and also latency is very high. It looks like any communication problem. Will this solve my problem or must produce adapter v3 ? I need to find a way how to integrate VRC700/6 into openhab.

![image](https://user-images.githubusercontent.com/116869405/198890843-a75190f0-5c0e-4068-8484-19ca115ac0c6.png)

![image](https://user-images.githubusercontent.com/116869405/198890871-b538b7c3-8982-49f7-a5e6-da3664e94f60.png)

![image](https://user-images.githubusercontent.com/116869405/198890928-5d1cc50e-1f08-4a75-9e61-bec6b05c2480.png)

**Comment by GHolli** [comment]
Hi,

you are using a CP2102 USB-serial converter, right?
Hard to say what's wrong, do you have a logic analyser? This would reveal whether you have collisions on the bus, for instance.

Producing a adapter v3 on your own will be tricky, because I think the schematic is not released.

I was thinking about some solution using a v2 adapter and an additional microcontroller to synchronise the communication with the bus and relax the timing for the communication (as done with the v3 adapter). But so far I only did a proof of concept.

**Comment by Alda2308** [comment]
As a first step yes, but then I switched to direct connection to decrease a latency, but problem was that hardware wasn't detected. So I check schematics from adapter v2 and found that there is a error with resistor value for tx signal. So I used lower value and now communication works excelent.
Now I send my new PCB to production. For v3 schematics exists, but time will show me if I want go this way. If yes, I will create a pcb for standard components (not SMD)
